packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html

```diff
@@ -90,3 +90,25 @@
     <!--end of go back to dashboard option -->
 </div>
 <!-- End of Suceess Banner -->
+<!--legal section-->
+<div class="mt-4">
+    <small>
+        <span
+            data-translate="user_account_identity_documents_legal_info1"
+            data-translate-values="{
+                link1: $ctrl.LEGAL_LINK1,
+                link2: $ctrl.LEGAL_LINK2
+            }"
+        ></span>
+        <span
+            data-translate="user_account_identity_documents_legal_info2"
+        ></span>
```

Comment:
probably to remove, related to this one:
https://github.com/ovh/manager/pull/9255/files#r1188330329